.github/workflows/deploy.yml

```
@@ -0,0 +1,93 @@
+env:
+  GCP_PROJECT_ID: ${{ secrets.GCP_PROJECT_ID }}
+  GCR_HOSTNAME: gcr.io
+  DOMAIN: ${{ secrets.DOMAIN }}
+  IMAGE_TAG: ${{ github.sha }}
+
+jobs:
+  deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+    environment: demo
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)